submissions/kotlyar-andrey/dom/script.js

```diff
@@ -0,0 +1,237 @@
+      "Эти боссы находятся в соборе Анор Лондо. Попасть к ним можно только через парадный вход, охраняемый Королевскими стражами, в центральный зал, где они в свою очередь охраняют покои Гвиневер Принцессы Света.",
+    video: "vvqpaVSSblc",
+  },
+  {
+    id: 8,
+    name: "Ложе Хаоса",
+    image: "chaos.webp",
+    description:
+      "Ложе Хаоса создала Ведьма Изалита, когда пыталась воссоздать Первое Пламя, но, создав его, не смогла контролировать. Ложе поглотило её, а её детей превратило в ужасных мутантов. Можно встретить некоторых из них: Квилег Ведьма Хаоса, Прекрасная Госпожа, Квилана. Также Ложе стало своеобразным инкубатором для всевозможных демонов хаоса. Отсюда родом такие как: Демон Капра, Демон Прибежища, Демон-Телец и большое количество подобных им монстров.",
+    video: "tkjANPFfEXQ",
+  },
+  {
+    id: 9,
+    name: "Нагой Сит",
+    image: "seath.webp",
+    description:
+      "Сит предал свой род и помог Гвину и другим Повелителям уничтожить род драконов. Причина в том, что он родился с дефектом — у него не было чешуи бессмертия как у других драконов, а значит рано или поздно он должен был умереть. Позже он стал герцогом и частью королевской семьи, а также получил осколок великой души от Гвина.",
+    video: "Zbr6U1xujOI",
+  },
+  {
+    id: 10,
+    name: "Четыре Короля",
+    image: "kings.webp",
+    description:
+      "Желание силы и власти стало причиной их падения во тьму, в Бездну, через которую когда-то прошёл легендарный Арториас Путник Бездны. Тёмный изначальный змей Каас даровал Четырём королям силу Повелителя Тьмы, которая способна высасывать человечность. Жители Нового Лондо были принесены в жертву Бездне, не обретя покоя и став Призраками, рыцари же стали Тёмными духами на страже затопленных руин.",
+    video: "UgccH5a4QZE",
+  },
+  {
+    id: 11,
+    name: "Нито Повелитель Могил",
+    image: "nito.webp",
+    description:
+      "Нито — первый из мёртвых, повелитель смерти. На заре Эры Огня заполучил одну из Великих Душ и в войне против Драконов, Нито насылал миазмы болезней и смерти на своих врагов.",
+    video: "j8EUZKuPzAA",
+  },
+  {
+    id: 12,
+    name: "Гвин Повелитель Пепла",
+    image: "gwyn.webp",
+    description:
+      "Гвин был повелителем Света и отцом минимум четверых детей: первенца Солнца, Гвиневер, Гвиндолина и Филианоры. Йоршка, называющая себя сестрой Гвиндолина, также может быть ещё одной дочерью Гвина. Гвин и его дети были богами, однако первенец был позже лишен божественного статуса в наказание за заключение мира с Древними Драконами.",
+    video: "aM_OOcZVU9s",
+  },
+];
+
+/* Создание меню с добавлением прослушивания события нажатия */
+function createMenu() {
+  const menu = document.querySelector(".menu");
+
+  menu.innerHTML = bosses
+    .map(
+      (boss) =>
+        `<a href="#" class="menu__item${
+          completed.includes(boss.id) ? " completed" : ""
+        }" id="menu_boss_${boss.id}" data-bossid="${boss.id}">${boss.name}</a>`
+    )
+    .join("");
```

Comment:
`[].map().join()` replace with `[].reduce((acc, item) => {}, '')` one method instead of two)